cfme/networks/topology.py

```diff
@@ -0,0 +1,200 @@
+import re
+import time
+
+from navmazing import NavigateToAttribute
+
+from cfme.networks.topology_view import TopologyView
+from cfme.utils.appliance import Navigatable
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from wait_for import wait_for
+
+
+class Topology(Navigatable):
+    LEGENDS = '//kubernetes-topology-icon'
+    ELEMENTS = '//kubernetes-topology-graph//*[name()="g"]'
+    LINES = '//kubernetes-topology-graph//*[name()="lines_obj"]'
```

Comment:
**REQUIRED**
This xpaths should be parts of the respectful views.

Reply:
So it should be part of TopologyView?

Reply:
Yes, that's right.